typescript-audit-logs-example/views/send_events.ejs

```diff
@@ -6,115 +6,305 @@
+                                    <div class="event-form-input-group flex-column flex-start">
+                                        <div><label for="range-start">Range Start</label></div>
+                                        <div><input required name="range-start" class="text-input" id="range-start"
+                                                type="text" value=<%=rangeStart%> placeholder="ISO-8601 value for start
+                                            of the export range"></div>
+                                    </div>
+                                    <div class="event-form-input-group flex-column flex-start">
+                                        <div><label for="range-end">Range End</label></div>
+                                        <div><input required name="range-end" class="text-input" id="range-end"
+                                                type="text" value=<%=rangeEnd%> placeholder="ISO-8601 value for end of
+                                            the export range"></div>
+                                    </div>
+                                    <div class="event-form-input-group flex-column flex-start">
+                                        <div><label for="filter-actions">Actions</label></div>
+                                        <div><input name="filter-actions" class="text-input" id="filter-actions"
+                                                type="text"
+                                                placeholder="Optional comma-separated list of actions to filter on">
+                                        </div>
+                                    </div>
+                                    <div class="event-form-input-group flex-column flex-start">
+                                        <div><label for="filter-actors">Actors</label></div>
+                                        <div><input name="filter-actors" class="text-input" id="filter-actors"
+                                                type="text"
+                                                placeholder="Optional comma-separated list of actors to filter on">
+                                        </div>
+                                    </div>
+                                    <div class="event-form-input-group flex-column flex-start">
+                                        <div><label for="filter-targets">Targets</label></div>
+                                        <div><input name="filter-targets" class="text-input" id="filter-targets"
+                                                type="text"
+                                                placeholder="Optional comma-separated list of targets to filter on">
+                                        </div>
+                                    </div>
+                                </div>
+                                <div class="flex flex-start">
+                                    <div class="flex width-150px">
```

Comment:
Noticed that this width-150px class is causing some weird button sizing. Removing it seems to fix the issue. Could you just delete this class? 
![CleanShot 2023-03-02 at 12 40 15](https://user-images.githubusercontent.com/47123183/222535595-2d82312d-6cf6-440d-85db-92103bc43a48.gif)